addons/mail/static/tests/core/record.test.js

```diff
@@ -808,30 +809,45 @@ test("record.toData() is JSON stringified and can be reinserted as record", asyn
         static id = "id";
         id;
         names = Record.attr([]);
+        due_datetime = Record.attr(undefined, { type: "datetime" });
         messages = Record.many("Message");
+        team = Record.one("Team");
     }).register(localRegistry);
     (class Message extends Record {
         static id = "body";
         body = Record.attr("");
     }).register(localRegistry);
+    (class Team extends Record {
+        static id = "name";
+        name;
+    }).register(localRegistry);
     const store = await start();
     const p = store.Person.insert({
         id: 1,
+        due_datetime: "2024-08-28 10:19:44",
         names: ["John", "Marc"],
         messages: [{ body: "1" }, { body: "2" }],
+        team: "Discuss",
     });
     expect(p.names).toEqual(["John", "Marc"]);
     expect(p.messages.map((msg) => msg.body)).toEqual(["1", "2"]);
+    expect(p.team.name).toEqual("Discuss");
     expect(toRaw(store.Person.records[p.localId])).toBe(toRaw(p));
+    const previousDueDatetime = serializeDateTime(p.due_datetime);
```

Comment:
should assert value at this step (`"2024-08-28 10:19:44"`)

Reply:
I didn't assert value because I was afraid of HOOT environment not being good with timezone. I know this affects mostly mock server, but since window is fully patched this could also poses some issue, e.g. test runs well on runbot but not locally or vice-versa